pom.xml

```diff
@@ -126,7 +126,7 @@
     <apache.curator.version>4.2.0</apache.curator.version>
     <aws.amazonaws.version>1.11.815</aws.amazonaws.version>
     <!-- Pin this version to make the library compatible with netty 4.1.52.Final version -->
-    <awssdk.version>2.16.104</awssdk.version>
+    <awssdk.version>2.15.79</awssdk.version>
```

Comment:
interestingly enough, both this comment and version string was added in the same PR: https://github.com/Alluxio/alluxio/pull/17242/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R128-R129

have you consulted with @elega on how the original version string was obtained?

Reply:
The original version was the higest version we could use that made the code build.  Any higher netty version will break the alluxio build because it has interface changes. Downgrading the version is okay and does not make any issue, giving that none actually uses the feature.